utils/logistics/csvToJson.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const path = require('path');
+
+ const convertCSVtoJson = async () => {
+    const data = fs.readFileSync(path.join(__dirname, 'pinToStd.csv'), 'utf8');
```

Comment:
is this for city and std code mapping? Do we need a function to convert csv to JSON. We can directly save the file as json.

Reply:
Yeah, I have saved the json file only. This is the utility i have created for the CSV to JSON conversion which can be used if needed in future.

Reply:
File name to be a parameter.

Reply:
Done